src/07-projects/project-items.apib

```diff
@@ -120,4 +120,20 @@ Get details for a single project item.
                 + type: `product` (string)
                 + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
             + added_at: `2018-12-04T16:44:33+00:00` (string)
-            + updated_at: `2018-12-05T16:44:33+00:00` (string)
\ No newline at end of file
+            + updated_at: `2018-12-05T16:44:33+00:00` (string)
+
+
+### projectItems.setParticipantRates [POST /projectItems.setParticipantRates]
+
+Set participant user rates for timetracking project items, optionally for a single project
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + project_id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, optional)
```

Comment:
Q: is it necessary to have this inside `filter`? Could be just `project_id` (but nullable of course).

Reply:
> Q: is it necessary to have this inside `filter`? Could be just `project_id` (but nullable of course).

Not certain. if we foresee any other possible filters in the future it would make sense, otherwise - if it only the project will be used - then we can drop the filter and use just project_id.

As this documentation isn't released, though, perhaps we can just use project_id for now

Reply:
I guess once you moved the project_id outside of filter, you forgot to delete the `filter` field